pkg/apis/carto/v1alpha1/cluster_delivery_test.go

```diff
@@ -17,37 +17,36 @@ package v1alpha1_test
 import (
 	"reflect"
 
+	v1alpha12 "github.com/vmware-tanzu/cartographer/pkg/apis/carto/v1alpha1"
```

Comment:
oh weird, apparently a bunch of places got this replacement 🤔 is that coming from the generators included here? an IDE? iirc goland tries to be smart about these things and sometimes gets it wrong

Reply:
yeah it's a little annoying, as you say it was a folder move in goland which auto updated the package imports, will push a better version shortly